config/locales/en.yml

```diff
@@ -239,7 +242,10 @@ en:
         custom_select_error: Select yes if your basic care needs are being met at the moment
       dietary_requirements:
         title: Do you have any special dietary requirements?
-        hint: For example, you're a vegetarian or you have a food allergy.
+        hint: |
+          <p class="govuk-hint app-hint">For example, you're a vegetarian or you have a food allergy.</p>
```

Comment:
The hints will now be black test so I would recommend changing `govuk-hint app-hint` to `govuk-body` and removing the <br>

Reply:
actually need to double check this.